geomesa-hbase/geomesa-hbase-tools/bin/bootstrap-geomesa-hbase-aws.sh

```diff
@@ -58,8 +58,8 @@ echo "# Auto-registration for geomesa coprocessors ${NL}export CUSTOM_JAVA_OPTS=
 
 # Deploy the GeoMesa HBase distributed runtime to the HBase root directory
 if [[ "$ROOTDIR" = s3* ]]; then
-  aws s3 cp /opt/geomesa/dist/hbase/$DISTRIBUTED_JAR_NAME $ROOTDIR/lib/ && \
-  echo "Installed GeoMesa distributed runtime to $ROOTDIR/lib/"
+  aws s3 cp /opt/geomesa/dist/hbase/$DISTRIBUTED_JAR_NAME ${ROOTDIR}lib/ && \
```

Comment:
seems like rootdir might not always have a trailing slash - won't it depend on what you enter when you configure the cluster? could we add a check to standardize it?

Reply:
possibly a line like this would remove any trailing slash:
`ROOTDIR="${ROOTDIR#/}"`

Reply:
yeah, that seems to work

Reply:
Yeah, that's a good point. I'll add the fix.